packages/cli/src/commands/create-app/createApp.ts

```diff
@@ -112,10 +113,11 @@ export default async (cmd: Command): Promise<void> => {
       choices: ['PostgreSQL', 'SQLite'],
     },
   ];
-  const answers: Answers = await inquirer.prompt(questions);
+  
+  const answers: Answers = await mapInqueryAnswersFromCommanderOptions(questions, cmd);
```

Comment:
Could we do this in a separate PR? 😅

Reply:
It seems as if whenever we make a PR to our own dfds:master it gets auto-merged into any open PR from dfds:master to spotify:master. Any idea on what is causing this behavior?

Reply:
Yeah it's intended behaviour, PRs are not snapshots but a living diff of merging one branch into another. When you merge a PR towards master in your fork, that gets reflected in this PR, as it's from the master branch of your fork.

One way of opening the PR in this case would be to keep a feature branch around in your fork, and make the PR from that. If it's all already merged into your master, you could create a new branch and bring in only the changes that you want to create a PR from. Some more info here and in the surrounding docs: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork

Reply:
Thanks!